clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/util/id/DcosSpinnakerAppId.groovy

```diff
@@ -45,10 +49,41 @@ class DcosSpinnakerAppId {
      *         Deemed safe because backslashes are replaced with underscores.
      *         <p/>
      *         Ex: {@code acct_foo_bar}
-     * @see #getSafeRegion()
+     * @see #getSafeGroup()
+     */
+    public String getSafeCombinedGroup() {
+        unsafeCombinedGroup.replaceAll(MarathonPathId.PART_SEPARATOR, SAFE_REGION_SEPARATOR)
+    }
+
+    /**
+     * @return The canonical DC/OS "region" (a.k.a the full group path in which the marathon application lives)
```

Comment:
Yeah, I'm fine with doing that rather than trying to add it here too.  Could you log the JIRA for that?